templates/default/pages/[[...slug]].jsx

```diff
@@ -3,9 +3,10 @@ import { allDocuments } from "contentlayer/generated";
 import { useMDXComponent } from "next-contentlayer/hooks";
 
 import { MdxPage } from "../components/MDX";
+import { getPageData } from "../lib/getPageData";
 
-export default function Page({ body, ...rest }) {
-  const mdxComponent = useMDXComponent(body.code);
+export default function Page({ body, data, ...rest }) {
```

Comment:
What was `...rest` before this?

One general question: does a given page have access to its frontmatter as data in the page.

Reply:
`...rest` hasn't changed after adding `data` property. It's still just "the rest" of properties. In this case it holds all the properties off the corresponding JSON file created by Contentlayer for this page, apart from `body`.

> One general question: does a given page have access to its frontmatter as data in the page.
No, afaik it doesn't.